_data-prepper/pipelines/configuration/processors/parse-json.md

```diff
@@ -18,7 +18,7 @@ You can configure the `parse_json` processor with the following options.
 | Option | Required | Type | Description |
 | :--- | :--- | :--- | :--- | 
 | `source` | No | String | The field in the `event` that will be parsed. Default value is `message`. |
-| `destination` | No | String | The destination field of the parsed JSON. Defaults to the root of the `event`. Cannot be `""`, `/`, or any whitespace-only `string` because these are not valid `event` fields. |
+| `destination` | No | String | The destination field of the parsed JSON. Defaults to the root of the `event`. Cannot be `""`, `/`, or any white space-only `string` because these are not valid `event` fields. |
```

Comment:
```suggestion
| `destination` | No | String | The destination field of the parsed JSON. Defaults to the root of the `event`. Cannot be `""`, `/`, or any white-space-only `string` because these are not valid `event` fields. |
```